libmscore/repeatlist.cpp

```diff
@@ -309,13 +315,16 @@ void RepeatList::unwind()
 
                   // only unwind if section starts and ends with actual real measure
                   if (sectionStartMeasureBase && sectionEndMeasureBase) {
-                        unwindSection(reinterpret_cast<Measure*>(sectionStartMeasureBase), reinterpret_cast<Measure*>(sectionEndMeasureBase));
```

Comment:
I didn't know.  I'm adding this rule to https://musescore.org/en/developers-handbook/musescore-coding-rules